pyproject.toml

```diff
@@ -39,9 +39,9 @@ yattag = "*"
 webptools = "*"
 supervisor = "*"
 superlance = "*"
-pytest = "*"
 
 [tool.poetry.dev-dependencies]
```

Comment:
Since Poetry 1.2.0, dependency group specification has switched to a new format ([ref](https://python-poetry.org/docs/1.3/managing-dependencies/#dependency-groups)), while dev-dependencies is deprecated:

`[tool.poetry.group.dev.dependencies]` is the new equivalent of `[tool.poetry.dev-dependencies]`

Further, this makes it easier to mark the dev dependencies as optional by default:
```toml
[tool.poetry.group.dev]
optional = true
```

Reply:
oh dangit

right, okay, fine. Good call :V